src/index.test.ts

```diff
@@ -1,11 +1,72 @@
-import * as strongConfig from './index'
```

Comment:
why has this file (and all the below) changed? has nothing to do with PR description or title

Reply:
As mentioned on Slack, we need to merge in !20 first. After the rebase, this PR is ready for review

Reply:
ah ok, better put that in the PR description so reviewers don't need to switch between two communication streams (plus this wouldn't work with external contributors)

Reply:
I'm well aware. Since you're the only reviewer right now, I thought this works

Reply:
after having merged the previous PR, it seems github is doing a worse job than gitlab in dealing with PRs that rely on other PRs.

e.g. it doesn't allow you to rebase a PR on top of master from the UI

so i think if possible we should always try to start new features based off `master` (exceptions can happen when features build upon other features in previous PRs, but this isn't the case here)